docker/Dockerfile.alpine

```diff
@@ -0,0 +1,34 @@
+FROM alpine:3.14.3
```

Comment:
The idea that I would like to propose here is to introduce some docker images in the GitHub action to make the code reproducible in the local machine, and move these docker files used for internal stuff inside the `docker` directory to maintain order in case of this docker files will grow in numbers.

But, I can move it inside the con `contrib` direcotry

Reply:
Yeah, these aren't really top-level artefacts, rather we provide them for convenience. Especially if we plan to use these images primarily for CI testing, we should not expose them on the top-level.

Reply:
I agree I moved the docker directory inside the contr directory, and I moved all the docker filed inside this directory.

Maybe this can be wrong or not needed? I was thinking that maybe having all the docker-related stuff inside a directory can help to keep the stuff organized.

What do you think?